### ISSUE TITLE
Casing matrice : support bois/comosite complet ou menuiserie pour finish propre du produit

### ISSUE THREAD

**Issue by adriens** [adriens/temps-attente-matrix-led#36]
# :grey_question: Contexte

Nous sommes entrain de construire un produit complet mais actuellement le packaging/casing est assez brut : l'idée de cette issue est d'améliorer cela en proposant un casing qui : 

- Soit "beau"
- Permette de disposer la matrice sur une table de manière stable
- Permette de disposer une batterie à l'intérieur
- Finisse bien la chose afin que notre installation puisse être considérée comme une décoration

L'idée de le faire avec un bois d'essence locale serait sympa.

**Comment by adriens** [comment]
@josegoue993 a parlé de collgues en mesure de faire ce genre de chose avec prise de côté et/ou prépartion impression 3d et fichiers `stl` : tu as pu avoir des news @josegoue993 ?

**Comment by adriens** [comment]
De mon côté, je chercher si je peux pas trouver un menuisier pour nous faire un truc beau en bois.

**Comment by adriens** [comment]
Post [linkedIn](https://www.linkedin.com/posts/adrien-sales_dowhatyoulove-lovewhatyoudo-creativemindset-activity-7261275422091288576-xIQS?utm_source=share&utm_medium=member_desktop) : 

![image](https://github.com/user-attachments/assets/85276ae7-d7ef-456d-aa18-2fecfa060707)

**Comment by adriens** [comment]
Je chercher aussi sur mon réseau privé sur FB

**Comment by adriens** [comment]
:wave: @josegoue993 quelles sont les news de ton côté?

**Comment by adriens** [comment]
@josegoue993 , tu peux me répondre stp ?
@raphb3383 : tu peux demander à @josegoue993 de me donner les news stp ?

**Comment by josegoue993** [comment]
@adriens Désolé du délai de réponse. La demande de casing sur mesure a bien été transmise à la menuiserie Verlaguet. Les cotes et les caractéristiques attendus du casing (beau, tenue stable, protection à l'avant par plexiglass ou autre matériau) ont été joints à la demande. Une proposition avec devis est prévue pour la semaine prochaine. 

@raphb3383

**Comment by adriens** [comment]
> Une proposition avec devis est prévue pour la semaine prochaine.

Super intéressant, on pourra ajouter cela au rapport en gestion des coûts. De nôtre côté on a développé l'endpoint dédié IoT : il sera prochainement déployé :rocket: ... probablement la semaine prochaine

**Comment by josegoue993** [comment]
Bonjour @adriens, j'ai relancé ce matin, la menuiserie pour le devis. Ils m'ont demandé plus de photos concernant la matrice. Vu avec eux, il est plus simple pour eux qu'il garde la matrice afin qu'ils nous proposent quelque chose. J'ai donc déposé la matrice cet après-midi avec eux et je la récupère au plus tard vendredi. Dès que j'ai des infos, je vous tiens informer.

@raphb3383

**Comment by adriens** [comment]
Carrément c la meilleure option 
C cool, on va bien profiter de notre report de soutenance 
Pas sur qu'ils aient déjà géré ce genre de projets.... ce sera vraiment intéressant

**Comment by josegoue993** [comment]
@adriens, la menuiserie VERLAGUET m'a transmis le devis pour un casing en Kohu avec vitre en verre à l'avant. Le montant est quand même très élevé. Je vais me renseigner auprès d'autres menuisiers.
[DEVIS_VERLAGUET.pdf](https://github.com/user-attachments/files/17963193/DEVIS_VERLAGUET.pdf)
 @raphb3383

**Comment by adriens** [comment]
AH yes 46000 XPF... ouep, là c overkill

![image](https://github.com/user-attachments/assets/b0c7989e-8292-4467-8400-ecd9c11623c2)

**Comment by adriens** [comment]
Hello @josegoue993 @raphb3383 : des news pour le casgin fait à la mano en bois ?

**Comment by raphb3383** [comment]
Salut @adriens , 

Je ne pourrais pas te répondre à ce sujet, je laisse José rebondir sur ta question. Désolé.

Merci.

**Comment by adriens** [comment]
Yes, le jour de la soutenance, pensez à me rapporter le suppot plexi svp 🙏
